k8s/crd-samples/deployment-cstor-pool.yaml

```diff
@@ -42,7 +46,15 @@ spec:
           mountPath: /dev
         - name: tmp
           mountPath: /tmp
-          mountPropagation: Bidirectional
+#          mountPropagation: Bidirectional
```

Comment:
remove it if not used. or. add comments explaining this property & mention the exact reason "why its not required".

Reply:
Perhaps you can put some comments in this yaml.

Reply:
will mention in comments

Reply:
will remove

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas